src/base.css

```diff
@@ -164,6 +166,102 @@
 	--editedrec-bg: darkorange;
 }
 
+@media (prefers-color-scheme: dark) {
+	:root {
+		--text: #ddd;
+		--bg: #111;
+
+		--logo-bg: #1a5e9a;
+		--logo-active-bg: #c00;
+		--logo-text: white;
+
+		--tocnav-normal-text: #999;
+		--tocnav-normal-bg: var(--bg);
+		--tocnav-hover-text: var(--tocnav-normal-text);
+		--tocnav-hover-bg: #080808;
+		--tocnav-active-text: #f44;
+		--tocnav-active-bg: var(--tocnav-normal-bg);
+
+		--tocsidebar-text: var(--text);
+		--tocsidebar-bg: #080808;
+		--tocsidebar-shadow: rgba(255,255,255,.1);
+		--tocsidebar-heading-text: hsla(203,20%,40%,.7);
+
+		--toclink-text: var(--text);
+		--toclink-underline: #6af;
+		--toclink-visited-text: var(--toclink-text);
+		--toclink-visited-underline: #054572;
+
+		--heading-text: #8af;
+
+		--hr-text: var(--text);
+
+		--algo-border: #456;
+
+		--del-text: #f44;
+		--del-bg: transparent;
+		--ins-text: #4a4;
+		--ins-bg: transparent;
+
+		--a-normal-text: #6af;
+		--a-normal-underline: #555;
+		--a-visited-text: var(--a-normal-text);
+		--a-visited-underline: var(--a-normal-underline);
+		--a-hover-bg: rgba(25%, 25%, 25%, .2);
+		--a-active-text: #f44;
+		--a-active-underline: var(--a-active-text);
+
+		--blockquote-border: silver;
+		--blockquote-bg: #181818;
```

Comment:
Please check this background color against http://fantasai.inkedblade.net/style/design/w3c-restyle/2020/readme#amendment I don't think opaque gray works well when the blockquote is within a differently-colored box for the same reason that opaque white didn't work well in the light colors when you refactored them.